src/DoctrineJsonFunctionsBundle/Resources/config/postgres_functions.yml

```diff
@@ -0,0 +1,14 @@
+doctrine:
```

Comment:
The same here. I'd like to see this removed completely.
It's up to the framework user to configure this.